src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor_Misc.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
i don't see a need for a misc file, just for one symbol.  can you just put this in the main visitor.cs file?